src/DistributedMetadata/CatalogService.cpp

```diff
@@ -695,7 +711,7 @@ void CatalogService::mergeCatalog(const NodePtr & node, TableContainerPerNode sn
         }
 
         {
-            /// FIXME, if table definition changed, we will need update the storage inline
+            deleteTableStorageByName(p.second->database, p.second->name);
```

Comment:
If there is no table definition change, we shall not delete the table storage

Reply:
fixed

Reply:
yes